src/components/Categories.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable no-alert */
```

Comment:
- [ ] Kindly ensure you do not disable linters as they help us write cleaner code.